backend/pkg/logger/data/logger.go

```diff
@@ -0,0 +1,115 @@
+package data
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/HyperloopUPV-H8/h9-backend/pkg/abstraction"
+	"github.com/HyperloopUPV-H8/h9-backend/pkg/logger"
+	"github.com/HyperloopUPV-H8/h9-backend/pkg/transport/packet/data"
+)
+
+type ValueName string
+
+const (
+	name abstraction.LoggerName = "data"
+)
+
+type Logger struct {
+	running    bool
+	lock       sync.Mutex
+	valueFiles map[data.ValueName]*os.File
+}
+
+type DataRecord struct {
```

Comment:
```suggestion
type Record struct {
```
before it would be used like this: `data.DataRecord` and now it is like `data.Record`